go.mod

```diff
@@ -616,6 +616,7 @@ require (
 	github.com/natefinch/atomic v1.0.1
 	github.com/open-telemetry/opentelemetry-collector-contrib/processor/filterprocessor v0.87.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/vcenterreceiver v0.87.0
+	go.opentelemetry.io/collector/exporter/debugexporter v0.89.0
```

Comment:
There are lots of changes to the go.mod. Updating `go.opentelemetry.io/collector` could impact other components, so we normally update such OTel dependencies in one big batch whenever we need to upgrade to a new version.

I think the reason lots of modules got updated is that `debugexporter` is imported as v0.89.0 instead of v0.87.0. Would you mind resetting the changes to go.mod, and instead just adding this line manually:
```
go.opentelemetry.io/collector/exporter/debugexporter v0.87.0
```
In theory that should be the only necessary change to go.mod.